File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/duckdb/BoolType.scala
package dev.typr.foundations.docs.duckdb

import dev.typr.foundations.{DuckDbType, DuckDbTypes}

@SuppressWarnings(Array("unused"))
object BoolType:
  //start
  val boolType: DuckDbType[java.lang.Boolean] = DuckDbTypes.boolean_
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/sqlite/BoolType.java
package dev.typr.foundations.docs.sqlite;

import dev.typr.foundations.SqliteType;
import dev.typr.foundations.SqliteTypes;

@SuppressWarnings("unused")
public class BoolType {
  // start
  // SQLite has no BOOLEAN storage class — values land as INTEGER 0/1.
  SqliteType<Boolean> boolType = SqliteTypes.boolean_;
  // stop
}
